01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -20,7 +20,7 @@ function __construct($name, $age)
     }
 
     /**
-     * @param int $age
+     * @param $age int
```

Comment:
(Oops, clicked the wrong "add a line note" button.. should've clicked the one below)